lib/dependencies/HarmonyImportDependencyParserPlugin.js

```diff
@@ -131,10 +154,47 @@ module.exports = class HarmonyImportDependencyParserPlugin {
 				return true;
 			}
 		);
+		parser.hooks.binaryExpression.tap(
+			"HarmonyImportDependencyParserPlugin",
+			expression => {
+				if (!parser.scope.inGuardPosition || expression.operator !== "in")
+					return;
+				const leftPartEvaluated = parser.evaluateExpression(expression.left);
+				if (leftPartEvaluated.couldHaveSideEffects()) return;
+				const leftPart = leftPartEvaluated.asString();
+				if (!leftPart) return;
+
+				const rightPart = parser.evaluateExpression(expression.right);
+				if (!rightPart.isIdentifier()) return;
+
+				const rootInfo = rightPart.rootInfo;
+				if (
+					!rootInfo ||
+					!rootInfo.tagInfo ||
+					rootInfo.tagInfo.tag !== harmonySpecifierTag
+				)
+					return;
+				const settings = rootInfo.tagInfo.data;
+				const ids = settings.ids
+					.concat(rightPart.getMembers())
+					.concat([leftPart]);
+				parser.scope.guards.add(createGuard(ids));
```

Comment:
TODO add support for `identifier !=/!== falsy`. Since we support only `&&` operator this also possible guard.